DataFormats/Math/interface/approx_exp.h

```diff
@@ -122,7 +122,11 @@ end;
 
 // valid for -87.3365 < x < 88.7228
 template <int DEGREE>
-constexpr float unsafe_expf_impl(float x) {
+#ifdef CMS_UNDEFINED_SANITIZER
+__attribute__((no_sanitize("signed-integer-overflow")))
```

Comment:
Could we add comments explaining why this is done? (so that a curious reader would not have to GitHub archaeology)